docs/API.md

```diff
@@ -107,17 +107,23 @@ A `<Link>` can know when the route it links to is active and automatically apply
 
 #### Props
 ##### `to`
-The path to link to, e.g. `/users/123`.
-
-##### `query`
+Can be either a string or an object.
```

Comment:
More succinctly: You can get rid of this and just link to [`history`'s docs on locations](https://github.com/mjackson/history/blob/master/docs/Location.md).